### ISSUE TITLE
Microphone randomly stops working

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#1139]
The microphone randomly stops working when you start a call. From what I can tell, this never happens during a call. This can be fixed with a restart, but it happens somewhat frequently and gets quite annoying.

Affects: Windows, Mac, GNU/Linux.

**Comment by ghost** [comment]
Might be related to #1092? Unlike #1092, the second and third calls after starting up qTox worked. There isn't an obvious trigger for why this is happening.

**Comment by dubslow** [comment]
Frankly, there's a lot of issues with qTox's AV.

**Comment by ghost** [comment]
```
16:53:57 onCallTriggered
16:53:57 "Core: Starting new call with 0 without video"
16:53:57 "Core: AV ringing with 0 without video"
16:53:57 onAvRinging
16:54:05 "Core: AV start from 0 without video"
16:54:05 "Core: preparing call 0"
16:54:05 Audio:  "suscribing"
16:54:05 onAvStart
16:54:25 onHangupCallTriggered
16:54:25 enableCallButtons
16:54:25 "Core: hanging up call 0"
16:54:25 timer started!!
16:54:25 "Core: AV end from 0"
16:54:25 "Core: cleaning up call 0"
16:54:25 Audio:  "unsuscribing"
16:54:25 onAvEnd
16:54:25 enableCallButtons
```

It happened a few moments ago, this is from qtox.log.

**Comment by zetok** [comment]
Are [newest qTox builds](https://wiki.tox.chat/binaries) still affected?

If they are, it's possible that fix is included in #2094.

**Comment by zetok** [comment]
Oh, right, issue was opened by ghost.. is that you, @mannol?

**Comment by mannol** [comment]
Nope, thats not me.
Sunday, 23 August 2015, 10:58AM +02:00 from zetok notifications@github.com:

**Comment by towlie** [comment]
I think it is possible that https://github.com/tux3/qTox/pull/2094 fixes this. When I tested this PR I had issues with second calls. The one who hang up has no sound output anymore from calls (notification sounds still work) The hack he mentioned in this PR solved this.
Also I think this is also related to https://github.com/tux3/qTox/issues/1750

**Comment by zetok** [comment]
#2415 should help with it, and was merged. Is the issue fixed?

**Comment by antis81** [comment]
Closing… should be fixed upstream by #2509.
